conmon/conmon.c

```diff
@@ -152,11 +175,11 @@ int main(int argc, char *argv[])
 	/* Create the container */
 	if (terminal) {
 		snprintf(cmd, CMD_SIZE,
-			 "%s create %s --pid-file pidfile --console %s",
-			 runtime_path, cid, slname);
+			 "%s create %s %s --pid-file %s --console %s",
```

Comment:
That's really weird, I'm pretty sure I've built runc from master, I'll give another try tomorrow I guess. Thanks for clarifying.